river/river.go

```diff
@@ -140,6 +140,13 @@ func (r *River) parseSource() (map[string][]string, error) {
 
 	// first, check sources
 	for _, s := range r.c.Sources {
+		if len(s.Tables) > 1 {
+			for _, table := range s.Tables {
+				if table == "*" {
+					return nil, errors.Errorf("wildcard * not allowed for multiple tables")
```

Comment:
is not allowed

Reply:
wildcard * not allowed for multiple tables, right...?